cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/NetworkClientImpl.kt

```diff
@@ -183,12 +182,19 @@ private class NetworkClientImpl(
             .get()
             .build()
 
-        val response = httpClient.newCall(request).execute()
+//        val response = httpClient.newCall(request).execute()
```

Comment:
Can we remove the commented code?

Reply:
sure, but only after the manual test I'm working on right now